application/core/database/cycle-stat.go

```diff
@@ -0,0 +1,35 @@
+package database
+
+import (
+	"github.com/google/uuid"
+	"github.com/koki/randommatch/entity"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+)
+
+func CreateStat(cycle entity.MatchingCycle) (string, error) {
+	driver, err := Driver()
+	if err != nil {
+		return "", err
+	}
+	session := (*driver).NewSession(neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
+	defer session.Close()
+
+	uid, err := session.WriteTransaction(func(tx neo4j.Transaction) (interface{}, error) {
+		result, err := tx.Run("CREATE (n:Stat {uid: $id}) "+
+			"ON CREATE SET n += {creation_date: datetime({timezone: 'Z'}), last_update: datetime({timezone: 'Z'})} "+
```

Comment:
you do not need to do on create SET since you are already doing create so put all parameters directly in line 18